researchspace/researchspace
# Semantic tree input search dropdown

# What

Updated `SemanticTreeInput` component:
- moved the input search inside the dropdown
- click on text label toggle dropdown

# Video / Gif / Screenshot


https://github.com/researchspace/researchspace/assets/6029465/b07c3189-a45d-4a5d-8483-e3d60ca17092


# How To Test

- Go to templates or forms where `SemanticTreeInput` is used
- Open the dropdown and search for an element on the list
- Search and selection should work as usual
